addons/website_forum/static/src/js/follow_tags.js

```diff
@@ -0,0 +1,64 @@
+odoo.define('website_forum.follow_tags', function (require) {
+'use strict';
+
+var ajax = require('web.ajax');
+var animation = require('web_editor.snippets.animation');
+
+animation.registry.follow_tags = animation.Class.extend({
+    selector: ".o_tags_follow",
+    start: function(editable_mode) {
+        var self = this;
+        if (editable_mode) {
+            self.$target.find(".o_js_forum_tag_follow").off('mouseenter mouseleave');
+        }
+        self.is_user = self.$target.data('is_user') || false;
+        self.email = self.$target.data('email') || '';
+        self.$target.find(".o_js_forum_tag_follow").each(function() {
+            var tag_id = $(this).find('a').data('tag-id');
+            var follow = self.$target.find("div[data-ids=" + tag_id + "]").data('follow');
+            var is_follow = (follow == 'on')?true:false;
+            self.toggle_subscription(tag_id, is_follow, self.email);
+            self.$target.find('.tag_btn[data-id=' + tag_id + ']').on('click', function(event) {
+                self.on_click_tag($(this).attr('data-id'));
+            });
+        });
+
+        return;
+    },
+    on_click_tag: function(data_id) {
+        var self = this;
+        var $email = this.$target.find(".o_tag_follow_email[data-id=" + data_id + "]");
+
+        if ($email.length && !$email.val().match(/.+@.+/)) {
+            this.$target.addClass('has-error');
+            return false;
+        }
+
+        var email = $email.length ? $email.val() : false;
+        if (email || this.is_user) {
+            ajax.jsonRpc('/website_mail/follow', 'is_follow', {
+                'id': data_id,
+                'object': 'forum.tag',
+                'message_is_follower': this.$target.find("div[data-ids=" + data_id + "]").attr('data-follow') || "off",
+                'email': email,
+            }).then(function (res) {
+                self.toggle_subscription(data_id, res["is_follow"], email);
+                if (!res["is_session"]) {
+                    location.reload();
+                }
+            });
+        }
+    },
+    toggle_subscription: function(tag_id, follow, email) {
+        this.$target.find(".js_tag_follow_btn#" + tag_id).toggleClass("hidden", follow);
+        this.$target.find(".js_tag_unfollow_btn#" + tag_id).toggleClass("hidden", !follow);
+        this.$target.find("a[data-tag-id=" + tag_id + "]").toggleClass("label-success", follow);
+        this.$target.find("a[data-tag-id=" + tag_id + "]").toggleClass("label-default", !follow);
```

Comment:
this.$target.find("a[data-tag-id=" + tag_id + "]").toggleClass("label-success", follow)
     .toggleClass("label-default", !follow);